src/packages/client/src/__tests__/integration/happy/blog-env-mssql/test.ts

```diff
@@ -18,8 +19,9 @@ describe('blog-env-mssql', () => {
     await setupMSSQL(setupParams)
 
     await generateTestClient()
-    const { PrismaClient } = require('./node_modules/@prisma/client')
+    const { PrismaClient, Prisma } = require('./node_modules/@prisma/client')
 
+    PrismaHelpers = Prisma
```

Comment:
Why?

Reply:
Because the import of prisma is done in `beforeAll`. I'd like to improve that in our tests.